docs/fastdds/getting_started/simple_app/includes/task_subscriber.rst

```diff
@@ -0,0 +1,118 @@
+Write the Fast DDS subscriber
+"""""""""""""""""""""""""""""
+
+From the `src` directory in the workspace, execute the following command to download the HelloWorldPublisher.cpp file.
+
+.. code-block:: bash
+
+    wget -O HelloWorldSubscriber.cpp \
+        https://raw.githubusercontent.com/eProsima/Fast-RTPS-docs/master/code/Examples/C++/DDSHelloWorld/src/HelloWorldSubscriber.cpp
+
+Now you have the subscriber's source code. The application runs a subscriber until it receives 10 samples under the
+topic HelloWorldTopic. At this point the subscriber stops.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldSubscriber.cpp
+    :language: C++
+    :linenos:
+
+Examine the code
+****************
+
+As you have noticed, the source code to implement the subscriber is practically identical to the source code implemented
+by the publisher.
+Therefore, we will focus on the main differences between them, without explaining all the code again.
+
+Following the same structure as in the publisher explanation, we start with the includes of the C++ header files.
+In these, the files that include the publisher class are replaced by the subscriber class and the data writer class by
+the data reader class.
+
+*   Subscriber. Is the object responsible for the actual reception of the data.
+*   DataReader. It registers in the application the topic (TopicDescription) that identifies the data to be read and
+    accesses the data received by the subscriber.
+*   DataReaderListener. This is the listener assigned to the data reader.
+*   DataReaderQoS. Structure that defines the QoS of the DataReader.
+*   SampleInfo. Is the information that accompanies each sample that is ‘read’ or ‘taken.’
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldSubscriber.cpp
+    :language: C++
+    :lines: 22,29
+
+The next line defines the HelloWorldSubscriber class that implements a subscriber.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldSubscriber.cpp
+    :language: C++
+    :lines: 33
+
+The public constructor and destructor of the class is defined below. The fields of the class will be the participant,
+the subscriber, the topic, the data reader, and the data type.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldSubscriber.cpp
+    :language: C++
+    :lines: 37-61
+
+
+As for the private fields of the class, it is worth mentioning the implementation of the DataReader listener.
```

Comment:
All general comments in the publisher apply here, such as the public/private order